### DISCUSSION TITLE
HA has a new BASIC light after the latest updates of zwave-js and HA

### DISCUSSION THREAD

**Discussion by novaware-io** [zwave-js/zwave-js#6494] [Request Support / Investigate Issue] [answered]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

Since the latest updates I can see that some sensors devices now have an entity showing as light in HA.

The devices are:
- Fibaro Motion Sensor FGMS001
- Fibaro Door/Window Sensor FGK101
- Fibaro Flood Sensor FGFS101
- Fibaro Smoke Sensor FGSS001

Some are always ON, others always OFF.

Here the diagnostic for one of them: https://pastebin.com/V63Wigjv
 I can provide the other ones if needed.

Thanks

### Software versions

Driver Version: 12.2.3
Server Version: 1.33.0
Home Assistant version 2023.11

### Device information

- Fibaro Motion Sensor FGMS001
- Fibaro Door/Window Sensor FGK101
- Fibaro Flood Sensor FGFS101
- Fibaro Smoke Sensor FGSS001

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [ ] The log includes a re-interview of the problematic device _(if applicable)_.

- [ ] The log includes the problematic interaction with the device _(if applicable)_.

- [ ] I provided the node ID of the problematic device _(if applicable)_.

[a comment by an automated account is not shown]

**Reply by novaware-io** [reply]
I don't think my log is needed here, correct? it's more about the device configuration that's why I've pasted the device diagnostic, but when somebody reviews this issue, please let me know if a log is still needed, thanks

**Comment by asayler** [comment]
Adding some additional devices that show up with the new basic control value and probably shouldn't (most of them are sensors with no function to control):

- Zooz ZSE29 Motion Sensor
- Zooz ZEN31 RGBW Controller (adds a bunch of duplicate basic controls for each color channel)
- Ecolink PIR Motion Sensor v2.5
- Ecolink DWZWAVE25 Door Sensor
- Vision ZD2102-5 Door Sensor
- Sensative Strip Guard Door/Window Sensor

**Reply by Technodad** [reply]
I am seeing this behavior with three elderly switches type  “[GE - In-Wall Paddle Switch, 300S](https://devices.zwave-js.io/?jumpTo=0x0063:0x5257:0x3533:0.0)”. This seems to have shown up in the past couple of weeks. I’ve turned on driver logging per the instructions.

**Edit**: Added [zwavejs_2023-11-05.log](https://github.com/zwave-js/node-zwave-js/files/13260123/zwavejs_2023-11-05.log).

**Reply by AlCalzone** [reply]
@asayler the sensors probably use Basic CC to report state. Can you re-interview one or two of those, while making a [driver log](https://zwave-js.github.io/zwave-js-ui/#/troubleshooting/generating-logs?id=driver-logs), loglevel `debug` and attach it here as a file (drag & drop into the text field).

**Reply by kpine** [reply]
> I am seeing this behavior with three elderly switches type “[GE - In-Wall Paddle Switch, 300S](https://devices.zwave-js.io/?jumpTo=0x0063:0x5257:0x3533:0.0)”.

These Jasco/GE devices are already fixed in [v12.3.0](https://github.com/zwave-js/node-zwave-js/releases/tag/v12.3.0). You'll need to wait until the application you are using updates, and then the entities will become unavailable and you can delete them.

> * Vision ZD2102-5 Door Sensor

I have this device and it does not have any light entity. I would second the suggestion to re-interview. It just did a re-interview today to make sure I didn't have outdated state and still no entities.

**Comment by Idaho947** [comment]
I add :
- Aeotec motion multisensor 6 and 3 (ZW100 and 
- Fibaro smoke detector (FGSD002)

**Comment by AlCalzone** [comment]
Everyone who's experiencing this, please re-interview the affected devices. If the issue persists after the re-interview, I'd like to see a [driver log](https://zwave-js.github.io/zwave-js-ui/#/troubleshooting/generating-logs?id=driver-logs), on loglevel `debug`, attached here as a file (drag & drop into the text field).

**Comment by The00Dustin** [comment]
I have six [ZSENS930](https://devices.zwave-js.io/?jumpTo=0x0178:0x5253:0x5433:0.0)'s.  Four of them were showing the basic set and two of them were not.  At 21:30 UTC, I reinterviewed node 020, which was showing it, and it still showed it afterwards.  At 21:34 UTC, I reinterviewed node 022, which was not showing it, and then it started showing it afterwards.

I also have 1 [VCZ1](https://devices.zwave-js.io/?jumpTo=0x026e:0x5643:0x5a31:0.0) which is showing this, and I was able to reinterview it at 23:35 UTC (node 011).

All three of the reinterviews discussed above should be included in this log file:
[zwave-js-ui-store.zip](https://github.com/zwave-js/node-zwave-js/files/13326793/zwave-js-ui-store.zip)

**Comment by Technodad** [comment]
I re-interviewed my GE switches, and the Basic devices seem to be gone. Regards,Dave L.[Mobile]On Nov 7, 2023, at 10:28 AM, AlCalzone ***@***.***> wrote:﻿
Everyone who's experiencing this, please re-interview the affected devices. If the issue persists after the re-interview, I'd like to see a driver log, on loglevel debug, attached here as a file (drag & drop into the text field).

**Comment by novaware-io** [comment]
I confirm that by installing the latest updates and re-interviewing the device the BASIC light disappeared for me

Thank you, let me know if I shall close this now

**Comment by thecobra666** [comment]
Have the same issue with my Fibaro smoke sensors. After re-interview the issue stays.

**Comment by AlCalzone** [comment]
So far it looks like the issue is limited to sensors. I've checked the specification and for all reporting type devices (this includes multisensors), the spec is pretty clear:
> Basic Command Class MUST NOT be supported

At least my Multisensor 6 though does report support for the CC and responds when queried, which is where the entity seems to come from (at least after reporting motion once).

I think the solution would be forcibly marking Basic CC for all sensors as **unsupported**, and if they issue Basic commands or report support, treat this as **controlling** Basic CC.

**Comment by AlCalzone** [comment, marked as answer]
This is now tracked as a bug [here](https://github.com/zwave-js/node-zwave-js/issues/6516)

**Comment by thecobra666** [comment]
I'm still having this issue. even after re-interviewing it keeps the switch in HA. Fibaro Smoke sensor. 

Updated to latest zwave-js ui

**Reply by AlCalzone** [reply]
I've identified a few more bugs that cause this. Working on a release right now.

**Comment by raresdragan** [comment]
Debug info:
---
Core
2025.8.3

Supervisor
2025.08.3

Operating System
16.1

Frontend
20250811.1

Z-Wave JS UI
11.2.1
Z-Wave JS
15.12.0

One of my Fibaro motion sensor still has the BASIC light. It has an older firmware. Not sure if it matters.
I just excluded / re-included / re-interviewed today.

Manufacturer: Fibaro
Product: Motion Sensor (FGMS001)
Current Firmware: 2.7

It exposes a light.bedroom_motion_sensor_basic 
Which in Home assistant is turned on/off my motion.
The boolean sensors entities are not updated / changed by motion
Only the light entity. Weird.

**Reply by AlCalzone** [reply]
Please open a new issue with all the required information, especially the log of an interview and triggering the motion sensor.
